Gemfile.lock

```diff
@@ -1,15 +1,18 @@
-GIT
-  remote: https://github.com/bugsnag/maze-runner
-  revision: 9ceea798e4278c87e61c61ee91088921e7627011
+PATH
+  remote: ../maze-runner
   specs:
-    bugsnag-maze-runner (1.0.0)
-      cucumber (~> 3.1.0)
-      cucumber-expressions (= 5.0.15)
+    bugsnag-maze-runner (3.3.0)
```

Comment:
I think an up-to-date Gemfile.lock (from running `bundle install`) needs to be pushed, as this one references `bugsnag-maze-runner` which is not in the Gemfile